crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -442,6 +442,21 @@ pub fn derive_system_label(input: TokenStream) -> TokenStream {
     derive_label(input, &trait_path)
 }
 
+#[proc_macro_attribute]
+pub fn system_label(_attr: TokenStream, input: TokenStream) -> TokenStream {
+    let input = parse_macro_input!(input as syn::Item);
+    let mut trait_path = bevy_ecs_path();
+    trait_path.segments.push(format_ident!("schedule").into());
+    trait_path
+        .segments
+        .push(format_ident!("SystemLabel").into());
+    (quote! {
+        #[derive(#trait_path, Clone, Hash, Eq, PartialEq, Debug)]
```

Comment:
This is definitely more ergonomic, but it's really only useful for simple types, right? If I wanted to do this with something that has a custom `Debug`, for example, I couldn't use this macro.

Is this okay? Or would it be better to allow for an optional list of traits to opt out of (e.g. `#[system_label(Debug)]` or something)? I'm not sure how everyone else feels about this.

Reply:
I think in 99% of the cases, you will want to do what this proc macro does. This PR also doesn't remove the old way, which is perhaps more flexible in allowing custom implementations.

Reply:
Yeah, I agree that this covers 99% of use cases.